documentation/migrations/3.0.0.md

```diff
@@ -0,0 +1,56 @@
+Migrating to version 3.0.0
+==========================
+
+Since version `3.0.0` we have introduced a couple of non-backwards compatible changes. This guide is
+supposed to help you migrate to it if you have been using previous versions (< `3.0.0`).
```

Comment:
```suggestion
helps you migrate from previous versions (< `3.0.0`).
```

Reply:
Using stronger language.
The guide is awesome, and it **will help you migrate** :)